doc/release-notes/release-notes-bucash1.2.0.0.md

```diff
@@ -0,0 +1,139 @@
+Release Notes for Bitcoin Unlimited Cash Edition 1.2.0.0
+=========================================================
+
+Bitcoin Unlimited Cash Edition version 1.2.0.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a minor release version based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specification you could find here:
+
+https://github.com/Bitcoin-UAHF/spec/blob/master/uahf-technical-spec.md
+
+
+Upgrading
+---------
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes for older versions), then run the
+installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
+bitcoind/bitcoin-qt (on Linux).
+
+The first time you run this new version, your UTXO database will be converted
+to a new format. This step could take a variable amount of time that will depend
+on the performance of the hardware you are using.
+
+Downgrade
+---------
+
+In case you decide to downgrade from BUcash 1.2.0.0 to a previous version you
+will need to run the old release using `-reindex-chainstate` option so that the
+UTXO will be rebuild using the previous format.
+
+Main Changes
+------------
+
+The main changes of this release is the introduction of the cashaddr new address format.
+The specification for this new address encoding could be find [here](https://github.com/Bitcoin-UAHF/spec/blob/master/cashaddr.md).
+The old format continue to be supported and won't be deprecated, The old format will remain the default till January 14th, 2017, on
+that day the new format will be used as default new format automatically.
+A command line flag `-usecashaddr` and configuration parameter could be used to select which format to by default.
+
+Other notable changes:
+
+- Activate the new DAA by height instead of MTP (removal of `newdaaactivationtime` parameter)
+- Cash net magic became mandatory. Peers incoming connections using the old magic would be rejected.
+- Add limited support to BitPay Bitcoin Cash address [format](https://support.bitpay.com/hc/en-us/articles/115004671663-BitPay-s-Adopted-Conventions-for-Bitcoin-Cash-Addresses-URIs-and-Payment-Requests)
+- Coin cache improvements
```

Comment:
also add:
"satoshi" field added to any RPC call that returns an "amount".  The "satoshi" field is the amount in satoshis (as an integer).  This field makes scripting easier since script authors do not need to use "perfect" fraction libraries such as  financial decimal or binary coded decimal numbers.

Reply:
Add two RPC calls allow importing multiple watch-only addresses (importaddresses) and private keys (importprivatekeys) and supercede importaddress and importprivkey.  The wallet rescan required after an import can now take hours, causing the original commands to time out and render it very difficult to import many addresses or keys.  These new commands allow batching of multiple addresses or private keys, allowing all addresses and private keys to be imported in one step.  Also, the rescan operation occurs asynchronously, so the RPC command returns right away rather than timing out.  Your script can determine the status of the rescan by running "getinfo" and looking at the new "status" field.

Reply:
+ Add verbose "getstat" that returns a timestamp for reported statistics.  And report the correct statistics regardless of system time changes (on platforms that support CLOCK_MONOTONIC)

Reply:
* Change currency unit to BCH

Reply:
I found the latter addition a little bit verbose but I can live with it.

Reply:
WRT change currency, we already did that already in the prev release is just that we miss a few places, I don't think  it is worth mentioning.